FeedSX/screens/UniversalFeed/index.tsx

```diff
@@ -264,14 +441,17 @@ const UniversalFeed = () => {
         </View>
       )}
       {/* create post button section */}
-      <View style={styles.newPostButtonView}>
+      <TouchableOpacity
+        disabled={postUploading}
+        style={styles.newPostButtonView}
+        onPress={() => NavigationService.navigate('CreatePost')}>
```

Comment:
Name `CreatePost` should be imported from the screen names enum/constant file

Reply:
resolved